providers/provider.go

```diff
@@ -8,4 +8,5 @@ type Provider interface {
 	Download(id string, captionsType string) ([]byte, error)
 	GetProviderJob(id string) (*database.ProviderJob, error)
 	GetName() string
+	CancelJob(id string) (bool, error)
```

Comment:
Does this ``bool`` mean cancelled or cancellable? If it means canceled, isn't just the ``error`` enough?

Reply:
It's `cancellable`